App/Seller/Sources/Seller/UI/MainView.swift

```diff
@@ -2,33 +2,45 @@
 import SwiftUI
 
 struct MainView: View {
-
+    
     var body: some View {
         TabView {
             Group {
-                MyOffersView()
-                    .tabItem {
-                        Label("Moje oferty", systemImage: "list.bullet")
-                    }
-                   
-                MyTransactionsView()
+                MessagesView()
                     .tabItem {
-                        Label("Transakcje", systemImage: "dollarsign.square")
+                        TabBarIcon(imageName: "messageTabIcon", titleName: "Wiadomości")
                     }
-                   
-                MessagesView()
+                MyOrdersView()
                     .tabItem {
-                        Label("Wiadomości", systemImage: "message")
+                        TabBarIcon(imageName: "orderTabIcon", titleName: "Zamówienia")
                     }
-
-                MySettingsView()
+                MyAccountView()
                     .tabItem {
-                        Label("Ustawienia", systemImage: "slider.horizontal.3")
+                        TabBarIcon(imageName: "accountTabIcon", titleName: "Konto")
                     }
             }
         }
         .navigationBarBackButtonHidden(true)
     }
+    
+    private struct TabBarIcon: View {
```

Comment:
Maybe we should have some group in `Utilities` package for `CommonUI`? Then later we could reuse it in different packages.